python_ta/contracts/__init__.py

```diff
@@ -178,9 +178,14 @@ def _check_class_type_annotations(instance: Any) -> None:
                 f'{repr(value)} did not match type annotation for attribute "{attr}: {annotation}"')
 
 
-def _check_invariants(instance, rep_invariants: Set[str], global_scope: dict) -> None:
+def _check_invariants(instance, klass: type, global_scope: dict) -> None:
     """Check that the representation invariants for the instance are satisfied.
+
+    Precondition:
```

Comment:
Remove this precondition. Instead, use `getattr(klass, '__representation_invariants__', set())` to use a default value of an empty set.